Packs/Base/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Base",
     "description": "The base pack for Cortex XSOAR.",
     "support": "xsoar",
-    "currentVersion": "1.1.9",
+    "currentVersion": "1.1.10",
```

Comment:
@agamm you need to generate release notes file to the pack version. see the release notes folder in the pack.

Reply:
Yeah I did, I just missed adding it to git. Pushing now.